dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/views/StrainView.java

```diff
@@ -52,4 +59,47 @@ public void resetTranslate() {
 		super.resetTranslate();
 		updateStrain();
 	}
+	
+	/**
+	 * Sets the panning of the {@link StrainView}.
+	 * @param x the amount to pan on the x axis.
+	 * @param y the amount to pan on the y axis.
+	 */
+	public void setPan(double x, double y) {
+		resetZoom();
```

Comment:
It's not exactly restoring anymore, but the `setPan` from my branch is the one we want. It does not require the zoom to be reset :)